.github/workflows/e2e-detox-pr.yml

```diff
@@ -10,7 +12,7 @@ on:
 jobs:
```

Comment:
@mvitale1989 I updated the group to: "${{ github.workflow }}-${{ github.event.pull_request.number }}". I removed the `github.event.label.name` because if I added 2 labels: "E2E iOS tests for PR" and then "E2E iOS tests for PR (LBW 1)", it would cancel the 1st label/trigger.

Although, this could potentially cancel-in-progress if you start with "E2E iOS tests for PR" and then you add another unrelated label like "QA Review" or something, even though "QA Review" will not trigger the first job within this action.

Reply:
To my knowledge, the cancelation behaviour you described would happen with  `${{ github.workflow }}-${{ github.event.pull_request.number }}`. But it's also not an issue, since it's unlikely we'll run both of them simultaneously for the same PR

Reply:
Ah, looking back on the file it seems that the [`labeled` event](https://github.com/mattermost/mattermost-mobile/blob/main/.github/workflows/e2e-detox-pr.yml#L8) will be triggered by any Label, including `QA Review` as you mention.

So I still recommend adding `${{ github.event.label.name }}` to the group, or it would risk confusing users that add unrelated labels and see their tests canceled